content/en/docs/tasks/security/dns-cert/index.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Istio DNS Certificate Management
+description: Shows how to provision and manage DNS certificates in Istio.
+weight: 90
+keywords: [security,certificate]
+---
+
+{{< boilerplate experimental-feature-warning >}}
+
+By default, the DNS certificates used by the webhooks of Galley and the sidecar
+injector are provisioned and managed by Citadel, which is a large component
+that maintains its own signing key and also acts as a CA for Istio.
+This task shows how to provision and manage DNS certificates for Istio control
```

Comment:
Add a blank line, followed by:

In certain deployments, you may want to use your own certificate authority instead of Citadel. In those cases, Citadel ends up being used strictly for its DNS certificate provisioning functionality. Rather than having to deploy Citadel at all in this case, you can instead leverage Chiron, a lightweight component that signs certificates using the Kubernetes CA APIs without maintaining its own private key.

Reply:
Done.